### PR TITLE
Add context menu for windows

### PR DESCRIPTION
**Pull request by 0x6431346e**
## What changed?
Added an option in the settings to enable "Add to IPFS via Orion" in the context menu (only on Windows)

How it works:
- the menu runs `orion.exe --add myfile`
- after orion has started we check if the app was opened with the `--add` argument, if so, we add those files

Caveat: does not work when Orion is already running because of the single instance code. (I'm still trying to find a solution)

This also works in dev if you do `yarn start --add ./path-to-file` (make doesn't forward the extra args)

### CODE REVIEW

File: app/windows/Settings/Components/ConnectivityPanel.jsx [hunk 1]
```diff
@@ -128,6 +139,20 @@ class ConnectivityPanel extends React.Component {
     document.execCommand('copy')
   }
 
+  _handleContextMenuChange = () => {
```
Comment (koalalorenzo) [inline app/windows/Settings/Components/ConnectivityPanel.jsx]: Some comments can help understanding

**Comment by koalalorenzo** [comment]
We should definitively fix the fact that it doesn't allow you to add a file if the app is already running.

I guess we can easily work around by listening for something different. For example:
We could have a file in the application and we have the application listening to this file content changing. Then the context menu will add the path to this file (script or the app is launched with an option)

Something like that, or something even better.

**Comment by koalalorenzo** [comment]
Related to: #126

File: app/setup-add-requests.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { app } from 'electron'
```
Comment (koalalorenzo) [inline app/setup-add-requests.js]: move it into `app/lib` maybe

File: app/entry.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
-const path = require('path')
-require('./lib/report/node.js')
+import { join as pathJoin } from 'path'
+import './lib/report/node.js'
 
+import { checkForAddRequests } from './setup-add-requests'
+checkForAddRequests()
```
Comment (koalalorenzo) [inline app/entry.js]: add a small comment also here so in case we get over this code we will understand what this line is doing

**Comment by 0x6431346e** [comment]
Ooops, on it!

**Comment by 0x6431346e** [comment]
Tested on windows (build_unpacked) and it works now :)

**Comment by koalalorenzo** [comment]
There is a minor think to fix in the tests (add a line at the end of one file 😓 )

**Comment by koalalorenzo** [comment]
Tested on windows 10. Looks good to me <3

**Pull request merged**